builtins/safe-default-configuration.txt

```diff
@@ -243,3 +243,200 @@ math semantics
 - mathsize
 - scriptlevel
 
+
+// SVG 2
+// https://svgwg.org/svg2-draft/
+// as referenced here: https://html.spec.whatwg.org/#refsSVG
+
+// Document structure, https://svgwg.org/svg2-draft/struct.html
+svg svg
+- viewBox
+- preserveAspectRatio
+- transform
+svg g
+svg defs
+svg symbol
+- preserveAspectRatio
+- viewBox
+- refX
+- refY
+// Purposely omitted: svg use.
+svg switch
+svg title
+svg desc
+svg metadata
+
+// Paths, https://svgwg.org/svg2-draft/paths.html
+svg path
+- pathLength
+- d
+
+// Shapes, https://svgwg.org/svg2-draft/shapes.html
+svg rect
+- pathLength
+- x
+- y
+- width
+- height
+- rx
+- ry
+svg circle
+- pathLength
+- cx
+- cy
+- r
+svg ellipse
+- pathLength
+- cx
+- cy
+- rx
+- ry
+svg line
+- pathLength
+- x1
+- y1
+- x2
+- y2
+svg polyline
+- pathLength
+- points
+svg polygon
+- pathLength
+- points
+// Text, https://svgwg.org/svg2-draft/text.html
+svg text
+- lengthAdjust
+- x
+- y
+- dx
+- dy
+- rotate
+- textLength
+svg tspan
+- lengthAdjust
+- x
+- y
+- dx
+- dy
+- rotate
+- textLength
+svg textPath
+- lengthAdjust
+- textLength
+- path
+- href
```

Comment:
This should probably be disallowed?

Reply:
I think we don't disallow `href` per se but instead give them a [special pass for `javascript` URLs etc](https://wicg.github.io/sanitizer-api/#sanitization)